_schema.graphql

```diff
@@ -2103,6 +2103,17 @@ type buyerCounterOfferPayload {
   clientMutationId: String
 }
 
+input buyerRejectOfferInput {
```

Comment:
I manually added that to this schema file. How would I go about getting that generated?

Reply:
So whatever changes I had made manually to this file were overwritten the next time it was generated. I know this for a fact because (a) @sepans and @ashkan18 told me so, and (b) there is a comment above the `rejectReason` parameter in this input that I didn't manually add.

Reply:
i think the `Input` fields start with lower case. This is probably auto generated change since pre-commit of MP updates this file but worth making sure in the schema this is also lower case. You can confirm that in GraphiQL or insomnia.